packages/groqd-playground/src/components/JSONExplorer.styled.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+import { Box } from "@sanity/ui";
+
+export const Root = styled(Box)`
+  font-family: Menlo, monospace;
+  font-size: 0.9em;
+  position: relative;
+`;
+
+export const Label = styled.span`
+  color: #9d1fcd;
+  @media (prefers-color-scheme: dark) {
+    color: #d05afc;
+  }
+`;
+
+export const Key = styled.span`
+  color: #1e61cd;
+  @media (prefers-color-scheme: dark) {
+    color: #5998fc;
+  }
+`;
+export const Value = styled.span`
+  color: #967e1c;
+  @media (prefers-color-scheme: dark) {
+    color: #dbb931;
+  }
+`;
+
+export const LineItem = styled(Box)<{
+  depth: number;
+  pointer?: boolean;
+  hasError?: boolean;
+}>`
+  padding-left: ${({ depth }) => depth * DEPTH_SC}px;
+  border-radius: 4px;
+  cursor: ${({ pointer }) => (pointer ? "pointer" : "initial")};
+
+  background-color: ${({ hasError }) => (hasError ? "#ffe5ea" : "initial")};
+
+  &:hover {
+    background-color: ${({ hasError }) => (hasError ? undefined : "#e7e7e7")};
```

Comment:
does this disable the style or just output `background-color: ;`

Reply:
Just disables the style, as to not override the "error" background color!